### PR TITLE
chore: Update position-node to 1.0.0-rc.0

### PR DESCRIPTION
**Pull request by joebowbeer**
## What was changed

* Update to TypeScript SDK 1.0.0-rc.0
* Use TestWorkflowEnvironment
* Add jest test
* Add eslint and prettier
* Add .nvmrc

## Checklist

1. How was this tested:

* Followed directions in README
* Switched between Java and Node pendulums in UI
* Ran `npm t` to run jest test

2. Any docs updates needed? N/A

### CODE REVIEW

File: position-node/src/workflows.test.ts [hunk 1]
```diff
@@ -1,5 +1,69 @@
-describe("This is a simple test", () => {
-    test("check something", () => {
-        expect("FIXME").toEqual("FIXME");
+import { TestWorkflowEnvironment } from '@temporalio/testing';
+import { Worker, Runtime, DefaultLogger, LogEntry } from '@temporalio/worker';
+import { exitSignal, pendulum } from './workflows';
+import { v4 as uuid4 } from 'uuid';
+
+async function withWorker<R>(worker: Worker, fn: () => Promise<R>): Promise<R> {
```
Comment (bergundy) [inline position-node/src/workflows.test.ts:6]: This isn't required anymore, use `Worker.runUntil` instead please.

File: position-node/src/workflows.test.ts [hunk 2]
```diff
@@ -1,5 +1,69 @@
...
+        angleAccel: 0,
+        angleVelocity: 0,
+        dt: 0,
+        speed: 0,          
+      }],
+      workflowId: uuid4(),
+      taskQueue: 'test',
     });
+    await handle.signal(exitSignal)
+    const result = await handle.result()
```
Comment (bergundy) [inline position-node/src/workflows.test.ts:66]: nit: these lines are missing semicolons, might want to run prettier.

**Review by bergundy — APPROVED** [review]
Thanks @joebowbeer!

File: position-node/src/workflows.test.ts (hunk 2 above)
Comment (joebowbeer) [reply position-node/src/workflows.test.ts]: done

**Comment by joebowbeer** [comment]
PTAL

File: position-node/src/workflows.test.ts [hunk 3]
```diff
@@ -1,5 +1,115 @@
...
+  });
+  await worker.runUntil(async () => {
+    const handle = await workflowClient.start(pendulum, {
+      args: [gameInfo()],
+      workflowId: uuid4(),
+      taskQueue: 'test',
+    });
+    await handle.signal(exitSignal);
+    const result = await handle.result();
+    expect(result).toBeUndefined;
```
Comment (bergundy) [inline position-node/src/workflows.test.ts:68]: Is this not a method?

**Review by bergundy — APPROVED** [review]
Thanks @joebowbeer!

File: position-node/src/workflows.test.ts (hunk 1 above)
Comment (bergundy) [reply position-node/src/workflows.test.ts]: I must have forgotten to update it, will do

**Pull request merged**